rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -517,7 +517,7 @@ fn install_canister_fails_if_memory_capacity_exceeded() {
         )"#;
 
     let mut test = ExecutionTestBuilder::new()
-        .with_subnet_execution_memory(memory_capacity as i64)
+        .with_subnet_execution_memory(memory_capacity as u64)
```

Comment:
```suggestion
        .with_subnet_execution_memory(memory_capacity)
```
Can `memory_capacity` not be inferred to be `u64` in this case?

Reply:
It's inferred as `usize` because we do some operations with it and other `usize`s above.